.github/workflows/publish.yaml

```diff
@@ -14,4 +15,18 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: json-schema
-          destination_dir: ${{ env.GITHUB_REF_SLUG }}
\ No newline at end of file
+          destination_dir: ${{ env.GITHUB_REF_SLUG }}
+  publish-pypi:
+    if: startsWith(github.ref, 'refs/tags/stac-model-v')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2.2.2
+      with:
+        python-version: 3.10
+    - name: Install poetry
+      run: make poetry-install
+    - name: Build and publish
+      run: |
+        poetry publish --build
```

Comment:
Probably should be `make publish` or similar, and leave the logic grouped together in `make`.

Reply:
ok done added a publish command to Makefile